openslides/assignments/models.py

```diff
@@ -361,6 +389,7 @@ class AssignmentOption(RESTModelMixin, BaseOption):
         settings.AUTH_USER_MODEL,
         on_delete=models.CASCADE)
     vote_class = AssignmentVote
+    weight = models.IntegerField(default=0)
```

Comment:
Please move this model field one line above. `vote_class` is not a model field (it should have been be separated by one line, but this has not be done before).